generators/client/files-angular.js

```diff
@@ -123,9 +123,6 @@ const files = {
         {
             path: ANGULAR_DIR,
             templates: [
-                // entities
-                'entities/_index.ts',
-                'entities/_entity.module.ts',
```

Comment:
we can have an index.ts for each entity in its own folder and a parent index.ts here which exports the other index.ts files if required